### PR TITLE
Convert to web app

### PR DESCRIPTION
**Pull request by vlsulliv**

### CODE REVIEW

File: src/main/java/com/palehorsestudios/ard/Game.java [hunk 1]
```diff
@@ -131,9 +132,11 @@ String Fight(Player player, String option) {
     /**
      * Method to instigate player using their special power.
      */
-    void UsePower(Player player, String option) {
-        System.out.println("use " + option);
+    String UsePower(Player player, String option) {
+        StringBuilder vsb = new StringBuilder();
+        vsb.append("use " + option);
         player.useSpecialPower();
```
Comment (tuckerc) [inline src/main/java/com/palehorsestudios/ard/Game.java]: You should include the result of player.useSpecialPower() in the returned String.

File: src/main/java/com/palehorsestudios/ard/Game.java [hunk 2]
```diff
@@ -154,8 +157,9 @@ String Look(Player player, String option) {
     /**
      * Method to invoke unlock chest method
      */
-    void unlockChest(Player player) {
+    String unlockChest(Player player) {
         player.getCurrentRoom().unlockChest();
```
Comment (tuckerc) [inline src/main/java/com/palehorsestudios/ard/Game.java]: Need to return the result of player.getCurrentRoom().unlockChest() in the returned String. You will have to modify unlockChest().

File: src/main/java/com/palehorsestudios/ard/characters/Player.java [hunk 3]
```diff
@@ -50,13 +50,14 @@ public String pickUpItem(Item item) {
...
-    public void dropItem(Item item) {
+    public String dropItem(Item item) {
         if (itemsInventory.contains(item)) {
             itemsInventory.remove(item);
             currentRoom.addItem(item);
         } else {
-            System.out.println("Can't drop this item! It's not in player's item inventory!");
+            return "Can't drop this item! It's not in player's item inventory!";
         }
+        return toString();
```
Comment (tuckerc) [inline src/main/java/com/palehorsestudios/ard/characters/Player.java]: This is returning the Player.toString(). Use a StringBuilder here as well.

**Review by tuckerc — CHANGES_REQUESTED** [review]

File: src/main/java/com/palehorsestudios/ard/characters/Player.java [hunk 4]
```diff
@@ -50,13 +50,14 @@ public String pickUpItem(Item item) {
      * and add the item into the room item list
      * if input is not in Item Enum, throw Exception
      */
-    public void dropItem(Item item) {
+    public String dropItem(Item item) {
         if (itemsInventory.contains(item)) {
```
Comment (wonil-park) [inline src/main/java/com/palehorsestudios/ard/characters/Player.java:55]: if (true) does not return...

File: src/main/java/com/palehorsestudios/ard/characters/Player.java (hunk 3 above)
Comment (wonil-park) [inline src/main/java/com/palehorsestudios/ard/characters/Player.java]: I'm not sure what this returns.
Recommend using StringBuilder and append proper String, then return.

File: src/main/java/com/palehorsestudios/ard/util/ConsoleManager.java [hunk 5]
```diff
@@ -107,13 +107,13 @@ private static int getInput(List<List<String>> options) {
...
-
+    public static String gameExplanation() {
+        StringBuilder vsb = new StringBuilder();
         boolean navigateMenu = true;
         MenuTrieNode curr = menu;
         while (navigateMenu) {
-            System.out.println("<" + curr.getTitle() + ">");
-            System.out.println(curr.getDescription());
+            vsb.append("<").append(curr.getTitle()).append(">");
+            vsb.append(curr.getDescription());
```
Comment (wonil-park) [inline src/main/java/com/palehorsestudios/ard/util/ConsoleManager.java]: Probably needs a \n here.

File: src/main/java/com/palehorsestudios/ard/characters/Player.java (hunk 4 above)
Comment (tuckerc) [reply src/main/java/com/palehorsestudios/ard/characters/Player.java:55]: I would add something like vsb.append("Picked up " + item + ".") here.

**Review by wonil-park — APPROVED** [review]

**Review by tuckerc — CHANGES_REQUESTED** [review]

**Review by wonil-park — APPROVED** [review]

**Review by tuckerc — APPROVED** [review]

**Pull request merged**